Week2/homework/maartjes_work.js

```diff
@@ -44,4 +44,8 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+const earned = tasks.map(task => task.duration / 60)
+  .filter(task => task >= 2)
+  .reduce((sum, task) => sum += task * 16, 0);
+
+console.log('Maartje has earned €' + Math.round(earned * 100) / 100);
```

Comment:
I personally I think it would be a bit more clean if you would have put the method in the function and only console log earned.

Also, you could also consider .toFixed(2) instead of doing Math.round which is probably a bit less code and a bit more descriptive.